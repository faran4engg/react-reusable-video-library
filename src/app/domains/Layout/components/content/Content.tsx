import { FC } from 'react';
import { ContentProps } from './types';

const cls = {
  main: 'h-full overflow-y-auto mt-3',
  mainWrapper: 'container grid px-4 lg:px-4 mx-auto',
};

const Content: FC<ContentProps> = ({ children }) => (
  <main className={cls.main}>
    <div className={cls.mainWrapper}>{children}</div>
  </main>
);

export default Content;
