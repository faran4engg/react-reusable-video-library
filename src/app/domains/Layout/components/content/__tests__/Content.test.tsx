import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import Content from '../Content';

const renderer = createRenderer();

describe('<Content />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<Content>{() => <div />}</Content>);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
