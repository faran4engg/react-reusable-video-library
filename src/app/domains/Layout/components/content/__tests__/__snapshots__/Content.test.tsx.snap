// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> should render and match the snapshot 1`] = `
<main
  className="h-full overflow-y-auto mt-3"
>
  <div
    className="container grid px-4 lg:px-4 mx-auto"
  >
    [Function]
  </div>
</main>
`;
